kibble/api/bundles_test.go

```diff
@@ -111,3 +112,26 @@ func TestBundlesApiToModel(t *testing.T) {
 	assert.Equal(t, "this is my tagline", model.Tagline, "expect tagline")
 	assert.Equal(t, "Bundle description", model.Description, "expect description")
 }
+
+func TestBundleCustomFields(t *testing.T) {
+	itemIndex := make(models.ItemIndex)
+
+	serviceConfig := commonServiceConfig()
+
+	apiBundle := BundleV1{
+		ID:    456,
+		Title: "Bundle 456",
+		CustomFields: map[string]interface{}{
+			"string": "wololo",
+			"int":    9,
+			"bool":   false,
+		},
+	}
+
+	b := apiBundle.mapToModel(serviceConfig, itemIndex)
+
+	assert.Equal(t, b.CustomFields["string"], "wololo")
```

Comment:
nor does `t`

Reply:
`t` is fine here since it is a common practice with Go tests. Also, Go encourage short variable names.
https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
Ah ok, gotcha